ccm_web/server/src/main.ts

```diff
@@ -83,9 +85,11 @@ async function bootstrap (): Promise<void> {
     SwaggerModule.setup('swagger', app, document)
   }
 
+  if (!isDev) app.enableShutdownHooks(['SIGINT'])
```

Comment:
This and some some of the Docker artifact and `.sh` script changes allow us to shutdown gracefully in production with `pm1`, which sends `SIGINT` signals to the child processes int he cluster.

We could also set up the application to shutdown gracefully in development, but I think that would require changing how we use `npm` scripts to start things, which started to feel like too much, or out of scope. It's more important for production, but if it's desirable, I can open a separate issue.